apps/mobile/package.json

```diff
@@ -75,7 +75,8 @@
     "@scure/btc-signer": "1.6.0",
     "@segment/analytics-react-native": "2.21.0",
     "@segment/sovran-react-native": "1.1.3",
-    "@shopify/flash-list": "1.8.0",
+    "@sentry/react-native": "6.10.0",
+    "@shopify/flash-list": "1.7.6",
```

Comment:
This was changed from `npx expo install --fix` @pete-watters any concern with this change?

Reply:
If `1.8` is working lets use that but to get it working previously I also had to add some workaround code we should reverse also in [this PR](https://github.com/leather-io/mono/pull/1171/commits/4bd89a6d8a63e41824525738671103b563f25838#diff-3e8cb053e72e93c816b3ad3cf3d3f9255357d2d6b566f82a221b0c345e34df7f)

Those workarounds weren't cleared up properly on update. I'll push a commit here to:
1. Remove `"@shopify/flash-list>tslib": "2.4.0"` from `mono/package.json` `pnpm: { overrides`
2. Remove `tslib: path.resolve(__dirname, 'node_modules/tslib')` from `metro.config`